data/filter/wg21.py

```diff
@@ -228,10 +237,15 @@ def build_row(elems):
 
     header = pf.Null()
     width = 0
+    caption = pf.Null()
+    captionsFound = 0
     table.content.append(pf.HorizontalRule())
     for elem in table.content:
         if isinstance(elem, pf.Header):
             header, width = build_header(elem)
+        elif isinstance(elem, pf.BlockQuote):
+            caption = elem.content[0]
+            captionsFound = captionsFound + 1
```

Comment:
Hm, maybe we can get rid of `captionsFound`, and do something like:
```python
        elif isinstance(elem, pf.BlockQuote):
            if caption is not None:
                pf.debug("[Warning] The following caption is being ignored by a Tony Table:",
                         pf.stringify(elem))
            caption = elem.content[0]
```

Reply:
ok